### DISCUSSION TITLE
Maya-2020 freezing on 5 padded frame number while using usd and abc in aiStandIn

### DISCUSSION THREAD

**Discussion by Sudarshanhavale** [Autodesk/arnold-usd#1679] [Q&A] [answered]
Hello Guys,

I am facing a Maya freezing issue while I try to load USD/ABC caches through Arnold `aiStandIn's`, particularly on the frame numbers with 5 paddings (e.g.: 10001+). Also, note that It works fine on 4 padded frames. 

This test always fails on the below environment
```
maya-2020.4.0
arnold-7.1.1.1
mtoa-5.1.1.1
mayausd-0.17.0_202204010740-82e8fc3
python-2.7.18
pymel-1.2.0
```

The same test works fine on the below environment
```
maya-2022.0.0
arnold-7.1.1.1
mtoa-5.1.1.1
mayausd-0.23.0 
python-3.7.7
pymel-1.2.0
```

I also tried to manually fix the padding inside the USD caches but that also did not work So considering these findings I am assuming that it could be a bug so just wanted to if there are any fixes to make it work inside Maya 2020.

You can export the default sphere into USD/ABC for the frame range 9999-10002 and try to reproduce this at your end.

**Note:** There isn't any issue with a .ass sequence, that works fine on the 5-padded frame range.

Thanks,
Sudarshan

**Comment by cpichard** [comment]
Hi @Sudarshanhavale ,
It looks like an mtoa issue, how is the sequence defined in the aiStandIn ?
C

**Reply by sebastienblor** [reply, marked as answer]
Hi @Sudarshanhavale , 
indeed for mtoa issues, it's better to contact arnold support, or post on community forums. 

Since it's the same arnold & mtoa version between the use case that works and the one that fails, it would probably suggest that it's a maya issue that was solved between maya 2020 and maya 2022, although it could also be a problem with python2. MayaUSD is probably not involved, since mtoa didn't even support mayaUSD in that version. 

Maya 2020 is pretty old now, we don't support it anymore, and no longer release MtoA versions for it. The more recent version that supported Maya 2020 was MtoA 5.2.2.4, so maybe you can give it a try and see if the issue is still there ? Independantly of this issue, there's been a year of USD improvements and fixes between 5.1.1.1 and 5.2.2.4.

**Reply by Sudarshanhavale** [reply]
Hi @cpichard ,

We simply exported animation into USD and called that USD file in aiStandIn .dso parameter. Then turned ON the use file sequence on aiStandIn parameter.

**Reply by Sudarshanhavale** [reply]
Hi @sebastienblor ,

Yes, You can close this ticket as the issue is resolved in newer Maya versions. 
Thanks for your response. 

C
